testing/jormungandr-integration-tests/src/jormungandr/recovery.rs

```diff
@@ -157,6 +157,8 @@ pub fn test_node_recovers_kill_signal() {
         &jormungandr,
     );
     let snapshot_before = take_snapshot(&account_receiver, &jormungandr, new_utxo.clone());
+    // Wait before stopping so transactions are flushed to disk
+    std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
There is still a marginal race condition because the flush interval is exactly one second (correct me if I'm wrong @eugene-babichenko). Wait two seconds to be sure.

Reply:
We checked and by default is every 15ms.

Reply:
Oh, I missed part of the description and you have researched the interval better. Ignore this.